apis/launchers.md

```diff
@@ -59,35 +59,43 @@ Granite.Services.Application.set_progress.begin (0.2f);
 
 As you can see, the method `set_progress` takes a `double` value type and is a range between `0` and `1`: from 0% to 100%. As with badges, Don't forget that `set_progress_visible` must be `true` and `.begin` is required for asynchronous methods.
 
-## Static Quicklists
+## Actions
 
-Static quicklists do not involve writing any code or using any external dependencies.
+Actions are specific functions your app can perform without already being open; think of them as alternate and more specific entry points into your app. Actions appear in the context menu of your app icon in the Applications Menu and Dock, and are searchable by name from the Applications Menu.
 
-Static quicklists are stored within your `.desktop` file. These are so called "actions". You can define many actions in your desktop file that will always show as an action in the application menu as well as in the dock.
+Adding actions to a `.desktop` file does not involve writing any code or using any external dependencies, though your app needs a way to distinguish between actions, e.g. with command line flags.
 
-The format is as follows:
+To add an action, add it by unique ID to a new `Actions` line in your app's .desktop file: 
 
-```text
+```ini
+Actions=ActionID;
```

Comment:
I'm not sure this part is super clear about where the `Actions` line should go. It would probably help to imply more of the desktop file:

```suggestion
[Desktop Entry]
Name=Hello Again
[...]
Actions=ActionID;
```

Reply:
yeah good call